Libraries/src/Amazon.Lambda.Annotations/README.md

```diff
@@ -179,4 +179,13 @@ parameter to the `LambdaFunction` must be the event object and the event source
 * FromBody
     * Map method parameter to HTTP request body. If parameter is a complex type then request body will be assumed to be JSON and deserialized into the type.
 * FromServices
-    * Map method parameter to registered service in IServiceProvider
\ No newline at end of file
+    * Map method parameter to registered service in IServiceProvider
+
+
+## Project References
+
+In order to keep the Lambda Annotations library lightweight, we don't reference needed packages such as the `Amazon.Lambda.APIGatewayEvents` if you are using related attributes. This means that users will have to add the appropriate package references to their projects to ensure the library compiles properly.
```

Comment:
minor: but we reference "you" in first sentence, then "users" in the second. Think it'd be nice to agree.

Reply:
Updated entire section based on Irene's suggestion